cmd/up/exec_unix.go

```diff
@@ -7,22 +7,13 @@ import (
 	"context"
 	"os"
 	"os/exec"
-	"runtime"
 	"strings"
 	"syscall"
 )
 
+// GetCommandToExec for non-windows GOOS
 func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*exec.Cmd, error) {
-	var c *exec.Cmd
-	if runtime.GOOS != "windows" {
-		c = exec.Command("bash", "-c", strings.Join(cmd, " "))
-	} else {
-		binary, err := expandExecutableInCurrentDirectory(cmd[0], he.workdir)
-		if err != nil {
-			return nil, err
-		}
-		c = exec.Command(binary, cmd[1:]...)
-	}
+	c := exec.Command("bash", "-c", strings.Join(cmd, " "))
```

Comment:
I'd use something like `shellquote.Join` instead of `strings.Join`. We could have several issues with `strings.Join`, for example with backslash characters